src/pages/_document.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @next/next/no-document-import-in-page */
```

Comment:
I still don't know about that one. I don't get this warning

Reply:
This is the only error I receive. Here it is: 

![image](https://user-images.githubusercontent.com/14351733/223369297-f5ca0b24-4d61-42c4-8970-b8d612d91320.png)

The others are warnings.

Reply:
When running yarn next lint --no-cache I still get the same error.